repository.rst

```diff
@@ -143,6 +143,14 @@ returned by ``go env GOPATH``.
 | A value for ``sum`` may be found in the ``go.sum`` file or by running                                                 |
 | ``go mod download -json <module>@<version>``.                                                                         |
 +--------------------------------+----------------------+---------------------------------------------------------------+
+| :param:`build_naming_convention`  | :type:`string`       | :value:`""`                                                |
```

Comment:
Make sure the `|` separators line up with the `+` above and below.

Default value should be `"import_alias"`.